app/views/idv/inherited_proofing/no_information.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render(
+      'idv/shared/error',
+      type: :warning,
+      title: t('inherited_proofing.errors.cannot_retrieve.title'),
+      heading: t('inherited_proofing.errors.cannot_retrieve.heading'),
+    ) do %>
+  <p>
+     <%= t('inherited_proofing.errors.cannot_retrieve.info') %>
+  </p>
+
+  <%= link_to t('inherited_proofing.buttons.try_again'), root_url, class: 'usa-button usa-button--big usa-button--wide' %>
+
+  <%= render(
+        'shared/troubleshooting_options',
+        heading: t('components.troubleshooting_options.default_heading'),
+        options: [
+          {
+            url: 'https://www.va.gov/resources/managing-your-vagov-profile/',
+            text: t('inherited_proofing.troubleshooting.options.get_va_help'),
+            new_tab: true,
+          },
+        ].select(&:present?),
```

Comment:
There's only one option, and it's not conditional, so I don't think we need the filtering here.

```suggestion
        ],
```